Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -431,6 +437,23 @@ define([
             get : function() {
                 return this._hasAlphaChannel;
             }
+        },
+
+        /**
+         * A flag indicating whether this imagery provider should be able to pick features - if it's set to false,
+         * UrlTemplateImageryProvider#pickFeatures will always return undefined.
+         *
+         * @memberof UrlTemplateImageryProvider.prototype
+         * @type {Boolean}
+         * @default true
+         */
+        enablePickFeatures : {
```

Comment:
Actually, you may see test failures from this, due to the way the tests verify that these types implement the `ImageryProvider` interface.  If so, just ignore this comment.